### ISSUE TITLE
error when installing python-casacore with casacore from kern-3

### ISSUE THREAD

**Issue by SpheMakh** [casacore/python-casacore#124]
I get this error when I install python-casacore from PyPI after upgrading to kern-3

```
Step 4/5 : RUN pip install python-casacore==2.2.0
 ---> Running in 80da505a52e3
Collecting python-casacore==2.2.0
  Downloading python-casacore-2.2.0.tar.gz (97kB)
    Complete output from command python setup.py egg_info:
    /tmp/pip-build-42Fzpa/python-casacore/setup.py:64: UserWarning: Could not find a boost library
      warnings.warn("Could not find a boost library")
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-42Fzpa/python-casacore/setup.py", line 138, in <module>
        if 'casa' in library:
    TypeError: argument of type 'NoneType' is not iterable
    
    ----------------------------------------

```

**Comment by tammojan** [comment]
Thanks. This should have been fixed in the commit after the release ( d8e1f94cd00f42408a460fdc294f926778fcbaff ). Could you try that? If the patch (`setup.py` is unfortunately a lot of patchwork currently) works for you, we can roll out a 2.2.1 release.

**Comment by gijzelaerr** [comment]
FYI you can quickly verify such a thing using docker and KERN.

Replicate the problem:
```
FROM kernsuite/base:3
RUN docker-apt-install casacore-dev python-pip python-numpy python-setuptools libboost-python-dev libcfitsio3-dev git
RUN pip install python-casacore==2.2.0
```

Verify the fix:
```
FROM kernsuite/base:3
RUN docker-apt-install casacore-dev python-pip python-numpy python-setuptools libboost-python-dev libcfitsio3-dev git
RUN pip install git+git://github.com/casacore/python-casacore.git@d8e1f94cd00f42408a460fdc294f926778fcbaff
```

my conclusion: the problem has been solved.
